Project.toml

```diff
@@ -8,7 +8,9 @@ Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 DocumenterTools = "35a29f4d-8980-5a13-9543-d66fff28ecb8"
 Krylov = "ba0b0d4f-ebba-5204-a429-3ac8c609bfb7"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
Plots is a large and complicated package. It should not be included as a dependency. @dmaldona what do you think?

Reply:
It should not be included. That was a mistake addition, I must have forgotten I was in the project environment when I added it.

Reply:
Will you push another version without it?

Reply:
Yep